tasks/main.yml

```diff
@@ -12,6 +12,11 @@
 - include_tasks: initialize.yml
 - include_tasks: configure.yml
 
+# Otherwise users and db are created in default data dir location (Ubunut: /var/lib/postgresql/...)
+# instead of the custom `postgresql_data_dir`
+- name: Reset Ansible handlers to ensure users and db are created in data dir {{ postgresql_data_dir }}
+  meta: flush_handlers
```

Comment:
I'm not 100% sure why this is necessary (since it could cause other unintended side effects, I use flush_handlers sparingly).

If this is the first time the service is being configured, the next task would basically do the same thing, correct?

Reply:
This was based on https://github.com/geerlingguy/ansible-role-postgresql/pull/160 so I just had used what was there, and I have been using it locally. 

I will take a look tomorrow to see what it does without the `flush_handlers`

Reply:
Have you been able to verify whether this works without flush_handlers?

Reply:
Sorry, I hadn't gotten to it. I just did, and it appears to be working without running `flush_handlers`